src/adv.py

```diff
@@ -38,14 +49,70 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+name = input("What is your name?\n")
+player = Player(name, room["outside"])
+
+valid_directions = ('n', 's', 'e', 'w')
 
 # Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+while True:
+    # * Prints the current room name
+    current_room = player.current_room
+    print("\nCurrent Location:", current_room.name)
+
+    # * Prints the current description (the textwrap module might be useful here).
+    description = current_room.description
+    # print(textwrap.wrap(description, 70))
+    print(f"{description}\n")
+
+    # * Waits for user input and decides what to do.
+    direction = input("What would you like to do? \n").strip().lower().split()
+    
+    #
+    # If the user enters a cardinal direction, attempt to move to the room there.
+    
+    ###### if direction == "n":
+    ######     player.current_room = current_room.n_to
+
+    ###### elif direction == "s":
+    ######     player.current_room = current_room.s_to
+
+    ###### elif direction == "w":
+    ######     player.current_room = current_room.w_to
+
+    ###### elif direction == "e":
+    ######     player.current_room = current_room.e_to
+    
+    if direction[0].startswith(valid_directions):
```

Comment:
Since your direction variable holds info about both the direction the player wants to move in as well as the items they pick up, maybe consider using another variable name that aligns more with what it does.